data/hai/hai reveal 5 philosophers.txt

```diff
@@ -1276,84 +1276,129 @@ mission "Hai Reveal [B13] Freedom"
+			`	The blast door that the force now stands before is twelve meters high and wide, and scanners show it to be six meters deep. It is designed to be closed from this side, though, and has not been sealed well against being reopened. Soon it grinds part-way open, evidently sabotaged, and the operatives put on their goggles and pass through into the darkness of the next room.`
+			`	You could be forgiven for hesitating to follow them, but General Choot'k appears beside you and places one strong paw on your shoulder. "Come, now. We will discover if our task is finished, or impossible. It is time for bravery." He guides you in after the operatives, along with his own troops.`
 			choice
-				`	(Chat with the Elenctic citizens before you go.)`
-				`	(It's been a long day. Leave for your ship.)`
+				`	"Lead on."`
+					goto leadon
+				`	"Are you calling me a coward? If I had more time, I'd challenge you to a duel for that."`
+			`	"Heh heh," he laughs in a low voice. "Maybe soon."`
+			action
+				set "Hai Reveal [B13] Freedom: threatened Choot'k"
+			label leadon
+			`	Scanners show the room is a vast cavern, some sort of public space suitable for ten thousand people or more. It's pitch black when you enter, but as the last of the troops file in, a second blast door closes behind you. High roof lights snap on making the room as bright as day in the blink of an eye.`
+			`	You see before you a couple hundred pirates, held captive by a large crowd of heavily-armed civilians - all of which are hesitantly pointing guns at the operatives. It seems the citizens arranged a violent rebellion to coincide with the Republic's attack. Three unarmed civilians wearing distinctive hooded robes walk towards the operatives: one Hai, one human, and to your surprise one lizard-like Korath. The leader of the Syndicate operatives exits her suit and approaches them.`
+			`	The Hai speaks. "We are the leaders of the civilian government of Elenchus. Although we are grateful for your assistance in defeating our violent rulers, we will not accept your dominion either. We have heard of your civil war. Elenchus is an independent nation; and we intend to stay so unless you intend to force otherwise. I can, however, promise that unlike our overlords, we have no interest in anything other than peace, and we will send no military ships through the wormhole.`
+			`	"Captain <last>, we understand our former slave masters agreed to pay tribute to you. We will continue these payments out of gratitude for ending our servitude."`
```

Comment:
```suggestion
```

I would highly second/suggest just removing this line. Replacement per Grey's comment above can come later as another focused PR if approriate.